SecretSanta/src/SecretSanta.Business/GroupRepository.cs

```diff
@@ -46,4 +84,26 @@ public void Save(Group item)
             MockData.Groups[item.Id] = item;
         }
     }
+
+    public class AddAssignmentResult
+    {
+        public bool IsSuccess => string.IsNullOrWhiteSpace(ErrorMessage);
+        public string? ErrorMessage { get; }
+
+        private AddAssignmentResult(string? errorMessage)
+        {
+            ErrorMessage = errorMessage;
+        }
+
+        public static AddAssignmentResult UserNotFound()
+            => new AddAssignmentResult("User not found");
+        public static AddAssignmentResult Error(string? eMessage)
```

Comment:
nit: In general we avoid abbreviations unless they are well know.
```suggestion
        public static AddAssignmentResult Error(string? errorMessage)
```